runtime/binding-http/src/main/java/io/aklivity/zilla/runtime/binding/http/internal/stream/HttpClientFactory.java

```diff
@@ -1033,7 +1036,7 @@ private void onUpgradedOrClosed(
 
         private int availableSlotSize()
         {
-            return bufferPool.slotCapacity() - headerSlotLimit - Long.BYTES - Integer.BYTES;
+            return bufferPool.slotCapacity() - httpQueueSlotLimit - Long.BYTES - Integer.BYTES;
```

Comment:
Remind me why we subtract `Long.BYTES - Integer.BYTES` here. Is it still needed? If so, let's use the appropriate constant and if not, then let's remove it.

Reply:
Yeah it is a bug, seems like have a wrong mental model. Removing.